### DISCUSSION TITLE
Orca slicer start gcode?

### DISCUSSION THREAD

**Discussion by Rpm49** [bassamanator/Sovol-SV06-firmware#143] [General]
Hello, is it possible for someone to help me get all of your klipper goodness working in orca slicer? 

I tried using someones settings from facebook and kept the start command but even when i copy paste eli's prusa slicer start gcode, all it does is go back and forth and create a line in the middle of the bed. Thank you. Its's apparently a really nice profile so :')
![Kllipper](https://github.com/bassamanator/Sovol-SV06-firmware/assets/123909250/749dd9eb-2c99-4807-8b8f-2bdf4856bcfa)

**Comment by bassamanator** [comment]
Check the adjust your slicer section and use the PS start gcode as is, do not modify.

**Reply by Rpm49** [reply]
That was weird even prusa slicer did it so I think it was my klipper files so i tested before I passed out prusa slicer and it seemed fine for the cube. This is like my 9th install and whatever happened this time made it do that and I assumed it was orca slicer. I re-did the klipper files and we're good. 

While I have you :( 

Can you tell me exactly how to enable kamp?

Like specifically where to enable it and where to change the 0 to 1 and if i have to add things to my slicer. I know this coding isnt that much of my cup of tea but I'm struggling to find exactly where :(

**Reply by Rpm49** [reply]
Also, i didnt change the print_end at all, but when orca slicer ends, it just stays in the last spot, and keeps the bed and noxzzle on without moving away.
![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/123909250/a4f111b6-8956-4d70-ad27-562d2d0d439e)

**Reply by bassamanator** [reply]
> Can you tell me exactly how to enable kamp?

It's in the FAQs.

You need to add print end macro, check adjust your slicer.

**Reply by Rpm49** [reply]
Alrighty. I found kamp.

Question.
I see from importing the prusa slicer files I have a few here. What ones would you reccomend? I am looking for good speed and quality of course..

Also, I have an input shaper and want to enable that. CAn you reccomend me on how to add it to the printer config by chance?

With those 2 things solved I think I'm good. I just am losing so much time vs my sovol sv06Klipper bad I cant wrap my mind around good settings...

![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/123909250/1064602e-866a-4f9b-bf39-0b5ab3d721f1)

**Reply by dasb00ter** [reply]
Did u figure this out? I'm having trouble following this did you just switch to PS from orca?

**Reply by Rpm49** [reply]
What Part? Profile, Kamp, or input shaper.

I have my Sv06+'s on some hybrid of profiles from elseware but it's at a good enough state with kamp enabled.

For my Sv06 Klipper pad. I've got a profile that i started adjusting the speed and settings from the sovol slicer. getting it +- as close as I can but it can print the orca cube nicely (Just idk if its that fast yet but its looking fast)

I still am not sure how to do kamp yet, I thought I enabled the settings but I do have a print.cfg that's referencing sovols packs and His packs so today i might just make a copy and do the print.cfg as Bass's and see if that's good enought. I was worried about other settings not being used on the pad but this is so hard for being a coding noob.

**Reply by Rpm49** [reply]
I but the start gcode worked kinda but no kamp, but it didnt want to turn off so more research there.

**Reply by dasb00ter** [reply]
I'm also sv06+

1) so the start gcode put into orca slicer was the same as in Prusa Slicer?
2)I found a few profiles in printables. I think joescalon is one. 
3) I think I know where to look for the 0 to 1 setting but I'm not adverse to you telling me. 
4)I think there are other input shaping info discussions I'm using Btt pad 7 so I will look at that

Calling the print end macro as Bassamanator suggested didn't work?

**Reply by Rpm49** [reply]
I believe it was.

If it doesnt work, flip the Print_Start to Start_Print and same with the Print_End.

I lost my orca slicer settings as i tried to sync via account login so I'm re-doing it all now Ugh

![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/123909250/f4ece9cb-cd2d-4df9-a85f-6a7ef2a70edf)

M104 S0 ; Stops PS/SS from sending temp waits separately
M140 S0
M117
PRINT_START BED=[first_layer_bed_temperature] HOTEND=[first_layer_temperature[initial_extruder]]
PURGE_LINE

;; If you are passing variables to PRINT_START, comment out above line and uncomment the last 3 lines instead. (I already did this)
;; Guide: https://github.com/AndrewEllis93/Print-Tuning-Guide/blob/main/articles/passing_slicer_variables.md
;; Make sure the variable names match (for example if you use "EXTRUDER" instead of "HOTEND")

**Reply by klb331** [reply]
THANKS! This is just what I needed. 

I also added   BED_MESH_PROFILE LOAD=default  to make sure it loaded the bed mesh profile every time .
